opensrp-opd/src/main/java/org/smartregister/opd/repository/OpdDiagnosisAndTreatmentFormRepository.java

```diff
@@ -51,6 +51,11 @@ public boolean saveOrUpdate(@NonNull OpdDiagnosisAndTreatmentForm opdDiagnosisAn
         return rows != -1;
     }
 
+    @Override
+    public boolean save(OpdDiagnosisAndTreatmentForm opdDiagnosisAndTreatmentForm) {
+        return saveOrUpdate(opdDiagnosisAndTreatmentForm);
```

Comment:
Update this implementation to  contain the save & throw an exception in `saveOrUpdate`

Reply:
i think its the other way around